### PR TITLE
feat(csi-334): added redis-cluster support

### PR DESCRIPTION
**Pull request by geka-evk**

### CODE REVIEW

File: mojaloop/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Mojaloop Helm chart for Kubernetes
 name: mojaloop
-version: 16.2.0
-appVersion: "ml-api-adapter: v14.0.5; central-ledger: v17.6.0; account-lookup-service: v15.2.3; quoting-service: v15.7.0; central-settlement: v16.0.0; bulk-api-adapter: v17.0.0; transaction-requests-service: v14.1.2; simulator: v12.1.0; mojaloop-simulator: v15.0.0; sdk-scheme-adapter: v23.4.0; auth-service: v15.0.0; als-consent-oracle: v0.2.2; thirdparty-sdk: v15.1.1; ml-testing-toolkit: v17.0.0; ml-testing-toolkit-ui: v15.4.2;"
+version: 16.2.1
+appVersion: "ml-api-adapter: v14.0.5; central-ledger: v17.6.0; account-lookup-service: v15.4.0; quoting-service: v15.8.0; central-settlement: v16.0.0; bulk-api-adapter: v17.0.0; transaction-requests-service: v14.1.2; simulator: v12.1.0; mojaloop-simulator: v15.0.0; sdk-scheme-adapter: v23.4.0; auth-service: v15.0.0; als-consent-oracle: v0.2.2; thirdparty-sdk: v15.1.1; ml-testing-toolkit: v17.0.0; ml-testing-toolkit-ui: v15.4.2;"
```
Comment (vijayg10) [inline mojaloop/Chart.yaml:5]: I think its better to bump up the ALS version below
`  - name: account-lookup-service
    version: ">= 14.11.0"`

File: quoting-service/values.yaml [hunk 2]
```diff
@@ -573,9 +573,9 @@ quoting-service-handler:
       enabled: false
       type: redis
       proxyConfig:
-        host: redis
```
Comment (vijayg10) [inline quoting-service/values.yaml:576]: Don't you want to bump quoting service version like ALS?

File: mojaloop/Chart.yaml (hunk 1 above)
Comment (geka-evk) [reply mojaloop/Chart.yaml:5]: done

File: quoting-service/values.yaml (hunk 2 above)
Comment (geka-evk) [reply quoting-service/values.yaml:576]: but it's already set to a proper value:
image
image

Comment (vijayg10) [reply quoting-service/values.yaml:576]: Ok

**Review by vijayg10 — APPROVED** [review]

**Pull request merged**